openmetadata-ui/src/main/resources/ui/src/components/common/searchbar/Searchbar.tsx

```diff
@@ -22,6 +23,7 @@ type Props = {
   typingInterval?: number;
   placeholder?: string;
   label?: string;
+  removeMargin?: boolean;
```

Comment:
Instead of this flag we can pass className as a prop and use it here.

Reply:
Already tried that method it's not working for spacing classes, to prevent such a situation we shall make sure from next time we should not pass any spacing related classes directly to reusable components so we can adjust spacing as per requirement from wherever we use that component